preset/classical-ml/docker-compose.yaml

```diff
@@ -37,6 +37,54 @@ services:
         https_proxy: ${https_proxy}
         no_proxy: ''
       context: .
+      labels:
+        docs: classical_ml
+        org.opencontainers.image.title: "Intel® AI Tools Selector Preset Containers - Classical ML"
+        org.opencontainers.base.name: "ubuntu:22.04"
+        org.opencontainers.image.name: "intel/classical-ml"
+        org.opencontainers.image.version: ${IDP_VERSION:-2024.1.0}-py${PYTHON_VERSION:-3.10}
+        dependency.python: ${PYTHON_VERSION:-3.10}
+        dependency.python.pip: requirements.txt
+        dependency.apt.bzip2: true
+        dependency.apt.ca-certificates: true
+        dependency.apt.diffutils: true
+        dependency.apt.gcc: true
+        dependency.apt.git: true
+        dependency.apt.gzip: true
+        dependency.apt.make: true
+        dependency.apt.patch: true
+        dependency.apt.rsync: true
+        dependency.apt.unzip: true
+        dependency.apt.wget: true
+        dependency.apt.xz-utils: true
+        dependency.conda.jupyterlab: 4.1.8
+        dependency.conda.notebook: 7.1.3
+        dependency.conda.jupyterhub: 4.1.5
+        dependency.conda.jupyter-server-proxy: 4.1.2
+        dependency.conda.mako: '>=1.2.2'
+        dependency.conda.pyjwt: '>=2.4.0'
+        dependency.conda.cryptography: '>=42.0.5'
+        dependency.conda.nodejs: '>=20.12.2'
+        dependency.conda.aiohttp: '>=3.9.4'
+        dependency.conda.idna: '>=3.7'
+        dependency.conda.oauthlib: '>=3.2.2'
+        dependency.conda.dpnp: "${DPNP_VERSION}"
+        dependency.conda.numpy: "${NUMPY_VERSION}"
+        dependency.conda.python: "${PYTHON_VERSION}"
+        dependency.conda.scikit-learn-intelex: "${SKLEARNEX_VERSION}"
+        dependency.conda.xgboost: "${XGBOOST_VERSION}"
+        dependency.conda.modin-ray: "${MODIN_VERSION}"
```

Comment:
You need to apply a default here, and in the case of DPNP_VERSION, apply the correct variable from your build args.

Reply:
done